features/path_rules.feature

```diff
@@ -165,10 +165,10 @@ Feature: Path rules
     And the request path must be "/aaa/ccc"
 
   Scenario: An Ingress with prefix path rules should match each labels string prefix
-    (prefix /aaa does not match request /aaaccc)
```

Comment:
This is a valid prefix.

Reply:
According to the KEP, this is the expected behavior:

> A path element refers is the list of labels in the path split by the '/' separator. A request is a match for path p if every p is an element-wise prefix of p of the request path. Note that if the last element of the path is a substring of the last element in request path, it is not a match (e.g. /foo/bar matches /foo/bar/baz, but does not match /foo/barbaz)

https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20190125-ingress-api-group.md#prefix-match